.github/actions/teardown-rocm/action.yml

```diff
@@ -21,4 +21,4 @@ runs:
       if: always()
       shell: bash
       run: |
-        sudo chown -R "${USER}" "${GITHUB_WORKSPACE}"
+        chown -R "${USER}" "${GITHUB_WORKSPACE}"
```

Comment:
@pruthvistony I think we need `sudo` because the scenario we're addressing here is one where the permissions for the workspace have been changed from the GHA user to `jenkins`. However, the issue with using `sudo` here is that the GHA user does not have sudo permissions. I think the way to address this would be to run this inside a docker container since we can use `sudo` in there.